app/models/manageiq/providers/ansible_tower/shared/automation_manager/credential.rb

```diff
@@ -1,75 +1,18 @@
 module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::Credential
   extend ActiveSupport::Concern
 
+  include ProviderObjectMixin
+
   module ClassMethods
-    def create_in_provider(manager_id, params)
-      manager = ExtManagementSystem.find(manager_id)
-      credential = manager.with_provider_connection do |connection|
-        connection.api.credentials.create!(params)
+    def provider_collection(manager)
```

Comment:
Yeah, up to you. I'm just thinking for consistency reasons. So our future self does not have to switch between `provider_object(connection)` and `povider_collection(manager)`